xbmc/filesystem/CurlFile.h

```diff
@@ -209,5 +211,6 @@ namespace XFILE
       MAPHTTPHEADERS m_requestheaders;
 
       long            m_httpresponse;
+      bool            m_isshoutcast;
```

Comment:
Because changes like these are exactly what made kodi codebase a mess over time. Problems should be solved at proper layer.